backend/corpora/common/corpora_orm.py

```diff
@@ -104,21 +104,21 @@ class ProcessingState(enum.Enum):
 
 class ValidationState(enum.Enum):
     """
-    Enumerates DbProject validation states.
+    Enumerates DbCollection validation states.
 
     NOT_VALIDATED - Validation not performed yet.
-    VALID - Project is valid.
-    INVALID - Project is invalid.
+    VALID - Collection is valid.
+    INVALID - Collection is invalid.
     """
 
     NOT_VALIDATED = "Not validated"
     VALID = "Valid"
     INVALID = "Invalid"
 
 
-class ProjectLinkType(enum.Enum):
+class CollectionLinkType(enum.Enum):
```

Comment:
this change will require a database migration

Reply:
The enums are defined in the database using this name. You'll need to alter the enum name in the database for this work correctly.